core/chains/legacyevm/chain.go

```diff
@@ -393,6 +394,16 @@ func (c *chain) SendTx(ctx context.Context, from, to string, amount *big.Int, ba
 	return c.Transact(ctx, from, to, amount, balanceCheck)
 }
 
+func (c *chain) LatestHead(_ context.Context) (types.Head, error) {
+	latestChain := c.headTracker.LatestChain()
+
+	return types.Head{
+		Identifier: strconv.FormatUint(latestChain.ID, 10),
```

Comment:
Should we consider exposing the block number as well?

Reply:
I thought that this was the block number, I'll change this

Reply:
There are no comments about what this field means

Reply:
Oh k. Lets make sure that this is the block number

Reply:
There is a function called BlockNumber() in latestChain object

Reply:
updated